lib/block-supports/behaviors.php

```diff
@@ -84,17 +84,22 @@ function gutenberg_render_behaviors_support_lightbox( $block_content, $block ) {
 
 	$aria_label = __( 'Enlarge image', 'gutenberg' );
 
+	$processor->next_tag( 'img' );
 	$alt_attribute = $processor->get_attribute( 'alt' );
 
+	// An empty alt attribute `alt=""` is valid for decorative images.
 	if ( null !== $alt_attribute ) {
 		$alt_attribute = trim( $alt_attribute );
 	}
 
+	// It only makes sense to append the alt text to the button aria-label when the alt text is non-empty.
 	if ( $alt_attribute ) {
 		/* translators: %s: Image alt text. */
 		$aria_label = sprintf( __( 'Enlarge image: %s', 'gutenberg' ), $alt_attribute );
 	}
-	$content = $processor->get_updated_html();
+
+	$processor = new WP_HTML_Tag_Processor( $block_content );
+	$content   = $processor->get_updated_html();
```

Comment:
if we don't have any calls to `set_attribute()` or other functions that modify the content, then this line isn't necessary. it should be equivalent to setting `$content = $block_content;`

am I missing some lines where we set the `alt` attribute or aria label before this line?

Reply:
I'm not sure I understand how the processor is used in this file. If that's true, then why creting one more instance at line 111 and assigning it to `$z`? Is the content modified in any way before that point?

Reply:
> if we don't have any calls to set_attribute() or other functions that modify the content, then this line isn't necessary. it should be equivalent to setting $content = $block_content;
>
> am I missing some lines where we set the alt attribute or aria label before this line?

It looks like this line is no longer necessary and was from an earlier implementation — we can remove this.

> I'm not sure I understand how the processor is used in this file. If that's true, then why creating one more instance at line 111 and assigning it to $z? Is the content modified in any way before that point?

This new instance also looks like it is not necessary — it can be removed and we can reuse the first instance.